dask/dataframe/tests/test_groupby.py

```diff
@@ -2329,3 +2329,32 @@ def test_groupby_split_out_multiindex(split_out, column):
     )
 
     assert_eq(ddf_result, ddf_result_so1, check_index=False)
+
+
+def test_groupby_large_ints_exception_cudf():
+    cudf = pytest.importorskip("cudf")
+    dask_cudf = pytest.importorskip("dask_cudf")
+
+    max = np.iinfo(np.uint64).max
+    sqrt = max ** 0.5
+    series = pd.Series(np.concatenate([sqrt * np.arange(5), np.arange(35)])).astype(
+        "int64"
+    )
+    pdf = pd.DataFrame({"x": series, "z": np.arange(40), "y": np.arange(40)})
+    gdf = cudf.from_pandas(pdf)
+    gddf = dask_cudf.from_cudf(gdf, npartitions=2)
+    pddf = dd.from_pandas(pdf, npartitions=2)
+    gg = gddf.groupby("x").std().compute()
+    pg = pddf.groupby("x").std().compute()
+    assert_eq(gg, pg)
+
+
+def test_groupby_large_ints_exception_pandas():
+    max = np.iinfo(np.uint64).max
+    sqrt = max ** 0.5
+    series = pd.Series(np.concatenate([sqrt * np.arange(5), np.arange(35)])).astype(
+        "int64"
+    )
+    df = pd.DataFrame({"x": series, "z": np.arange(40), "y": np.arange(40)})
+    ddf = dd.from_pandas(df, npartitions=1)
+    ddf.groupby("x").std().compute(scheduler="single-threaded")
```

Comment:
Should we be testing this against some known result with `assert_eq`, or is computing it sufficient?

Reply:
Hmm yes i think I can compare it with `dd.groupby"x").std()`